src/pages/game/is-guessed-toast/isGuessedToastSlice.ts

```diff
@@ -0,0 +1,56 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../../../app/store';
+import { isGuessedToastSliceType } from '../../../types/isGuessedToastSlice';
+import { POLISH_COMMON_WORDS } from '../../../utils/constants';
+import { setGuessText } from '../guess-bar/guessSlice';
+
+const initialState: isGuessedToastSliceType = {
+    guessedWords: [],
```

Comment:
we already have those words in guessHistorySlice lets not duplicate it

Reply:
I guess you will encounter obstacle when trying to use state from guessHistorySlice. In that case you can either combine those states and keep all guess related stuff in guessSlice or use create selector in which you will make the actual check - this way you would have to make the check before setGuessText.
I think the first option is better as we are keeping more logic outside of react. Moreover later on we will probably have more guess related logic which requires cross state data.